src/utils.ts

```diff
@@ -1433,3 +1435,61 @@ export async function checkCompatibilityNativePackages(supabase: SupabaseClient<
     localDependencies: nativePackages,
   }
 }
+
+export async function promptAndSyncCapacitor(
+  isInit?: boolean,
+  orgId?: string,
+  apikey?: string,
+): Promise<void> {
+  // Ask user if they want to sync with Capacitor
+  const shouldSync = await pConfirm({
+    message: 'Would you like to sync your project with Capacitor now? This is recommended to ensure encrypted updates work properly.',
+  })
+
+  // Handle user cancellation
+  if (isCancel(shouldSync)) {
+    // For init flow, mark the cancellation
+    if (isInit && orgId && apikey) {
+      await markSnag('onboarding-v2', orgId, apikey, 'canceled', '🤷')
+    }
+    // Always exit on cancellation
+    exit()
```

Comment:
we dont exist anymore please throw instead

Reply:
this is required for our testing framework and SDK feature

Reply:
@riderx There are still some places where we `exit()` instead of throwing. I will fix this for my code